docs/concepts/providers.mdx

```diff
@@ -9,16 +9,14 @@ type: 'docs'
 
 ## Overview
 
-**Providers** are executable host plugins dedicated to longer-lived processes fulfilling non-functional requirements (sometimes called **capabilities**). Providers are typically responsible for functionality that is not considered part of the core business logic of an application, such as...
+**Providers** are executable [host](/docs/concepts/hosts) plugins dedicated to longer-lived processes fulfilling non-functional requirements (sometimes called **capabilities**). Providers are typically responsible for functionality that is not considered part of the core business logic of an application, such as...
 
 - Sending notifications
 - Fetching secret values
 - Accessing databases
 - Serving content over HTTP
 
-For example, when writing a web service that exposes a [RESTful API](https://en.wikipedia.org/wiki/REST), the HTTP server and the database are not parts of your business logic&mdash;your primary concern is structuring your API so that users can easily get the data they need. If you were to write this web service in a component, you would depend on providers to fulfill HTTP and database functionality in order to execute your business logic.
-
-In wasmCloud, providers are self-contained processes that implement a well-defined API. Providers run on a wasmCloud lattice and ensure that functionality depended on by components is available.
+For example, when writing a web service that exposes a [RESTful API](https://en.wikipedia.org/wiki/REST), the HTTP server and the database are not parts of your business logic&mdash;your primary concern is structuring your API so that users can easily get the data they need. If you were to write this web service in a component, you would depend on providers to fulfill HTTP and database functionality in order to execute your business logic. In wasmCloud, providers are self-contained processes that implement a well-defined API.
```

Comment:
```suggestion
**Components that contain business logic use the capabilities offered by providers.**

For example, when writing a web service that exposes a [RESTful API](https://en.wikipedia.org/wiki/REST), the HTTP server and the database are not parts of your business logic&mdash;your primary concern is structuring your API so that users can easily get the data they need. If you were to write this web service in a component, you would depend on providers for non-business-logic capabilities:

 - HTTP server (to serve HTTP requests)
 - Database access (to save changes made via the interface)
 
In wasmCloud, providers implement APIs (WIT interfaces) that enable functionality like the case described above.
```